### PR TITLE
WIP: add links to dummy file so tests can stay green

### PR DESCRIPTION
**Pull request by emilyville**
add a dummy page and send all not real links to it for now

For Dummy file, still need to do something like put it in a _dev-only directory and enable it in `_config.yml` and then disable it in `_config_production.yml

### CODE REVIEW

**Comment by emilyville** [comment]
@awong-dev I have no idea if what I added to production config will work, the internet told me it would.

File: dummy-placeholder.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  <div class="section main-menu">
+    <div class="row">
+      <div class="small-12 columns">
+        <h3>DUMMY PAGE</h3>
+      </div>
+    </div>
+  </div>
+
+<div class="section one">
```
Comment (awong-dev) [inline dummy-placeholder.html]: erp?  why bother with all this stuff?

File: dummy-placeholder.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+layout: home
+body_class: fourohfour
+title: Vets.gov
+---
+
+<div class="main home" role="main">
+  <div cl
```
Comment (awong-dev) [inline dummy-placeholder.html]: random newline?

File: _config_production.yml [hunk 3]
```diff
@@ -1,2 +1,4 @@
 domain: www.vets.gov
 destination:  ./_site_production
+
+exclude: [dummy-placeholder.html]
```
Comment (awong-dev) [inline _config_production.yml]: Hmm... so this doesn't actually work the way I was thinking it would.

Comment (awong-dev) [inline _config_production.yml]: We need to do it backwards for now...make the file default hidden...explicitly include it in `_config.yml` and then remove that `include` in `_config.production.yml` (which only works cause we don't use `include`.

File: _employment/job-seekers/unemployment-support.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ You will need the following documents when you apply:
 - Social security card 
 - Current résumé or military job history 
 
-If you are unemployed and at risk of becoming homeless, [VA can help](vets.gov homeless section). 
+If you are unemployed and at risk of becoming homeless, [VA can help]({{site.baseurl}}/dummy-placeholder.html).
```
Comment (awong-dev) [inline _employment/job-seekers/unemployment-support.md]: Do you need the {{site.baseurl}} everywhere?  These are just server-relative links so I think you can do `/_dummy-placeholder.html`.

**Comment by awong-dev** [comment]
Made some changes in 3f5f5ca.

**Comment by emilyville** [comment]
Yay! Lgtm

**Pull request merged**